database/migrations/2021_09_24_105720_add_circle_verification.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use App\Models\Circle;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddCircleVerification extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('circles', function (Blueprint $table) {
+            $table->boolean('is_verified')->default(0);
+        });
+
+        DB::table('circles')->update(['is_verified'=> 1]);
```

Comment:
So this will mark all old circles as `is_verified = true` after we run this migration, correct?

Reply:
thats right